.travis.yml

```diff
@@ -2,14 +2,15 @@ language: go
 go:
 - 1.5.1
 - tip
+before_install:
+  - rvm install 2.2.3
+  - nvm install 5.5
 install:
 - go get -u -v github.com/spf13/hugo
-- gem install scss_lint
 - npm install
 script:
-- gulp
 - npm test
-- gulp production build:website
+- npm run build
```

Comment:
Travis shouldn't know about gulp so now we're using `npm-run` to manage script commands on Travis.